README.md

```diff
@@ -101,312 +39,81 @@ terraform {
-| `instance_type`     | `m`, `l`, `xl`                           | `m`                                                                       | Sets the instance CPU size. You can also specify vendor specific machines in AWS i.e. `t2.micro`. [See equivalences](#Supported-vendors) table below.                                                                                                                                            |
-| `instance_gpu`      | ``, `tesla`, `k80`                       | ``                                                                        | Selects the desired GPU for supported `instance_types`.                                                                                                                                                                                                                                          |
-| `ssh_private`       |                                          |                                                                           | An SSH private key in PEM format. If not provided, one private and public key wll be automatically generated and returned in `terraform.tfstate`                                                                                                                                                 |
-
-### Machine
-
-Setup instructions:
-
-1. Setup your provider credentials as ENV variables
-
-<details>
-<summary>AWS</summary>
-<p>
-
-```sh
-export AWS_SECRET_ACCESS_KEY=YOUR_KEY
-export AWS_ACCESS_KEY_ID=YOUR_ID
+```console
+git clone https://github.com/iterative/terraform-provider-iterative
+cd terraform-provider-iterative
 ```
 
-</p>
-</details>
+### Install the provider
 
-<details>
-<summary>Azure</summary>
-<p>
+Build the provider and install the resulting binary to the [local mirror directory](https://www.terraform.io/docs/cli/config/config-file.html#implied-local-mirror-directories):
 
-```sh
-export AZURE_CLIENT_ID=YOUR_ID
-export AZURE_CLIENT_SECRET=YOUR_SECRET
-export AZURE_SUBSCRIPTION_ID=YOUR_SUBSCRIPTION_ID
-export AZURE_TENANT_ID=YOUR_TENANT_ID
+```console
+make install
 ```
 
-</p>
-</details>
+### Create a test file
 
-2. Save your terraform file `main.tf`
+Create a file named `main.tf` in an empty directory with the following contents:
 
-<details>
-<summary>AWS</summary>
-<p>
-
-```tf
+```hcl
 terraform {
   required_providers {
     iterative = {
-      source = "iterative/iterative"
+      source = "github.com/iterative/iterative"
     }
   }
 }
 
 provider "iterative" {}
 
-resource "iterative_machine" "machine" {
-  cloud = "aws"
-  region = "us-west"
-  name = "machine"
-  instance_hdd_size = "10"
-  instance_type = "m"
-  # Uncomment it if GPU is needed:
-  # instance_gpu = "v100"
-}
+# ... paste resource blocks from
+# ... the documentation examples
```

Comment:
...or not. Do we really want to maintain another full code snippet just to exemplify prepending `github.com` to the name?

Reply:
oh if this was a user-facing instruction rather than an incomplete note-to-self, then...

```suggestion
# ... other resource blocks ...
```

or

```suggestion
# ... etc ...
```

or even

```suggestion
```

Reply:
```suggestion
# ...
```
Even this, maybe?